### DISCUSSION TITLE
Replies to comments

### DISCUSSION THREAD

**Discussion by arpit0515** [dolanmiu/docx#1950] [Q&A] [answered]
I have a use case where my angular application needs to export replies along with comments mentioned. Earlier comments were being handled by Comments, now it is CommentRangeStart() and CommentRangeEnd(). What about replies? How to handle replies and what are the functions to be used for replies?

**Comment by dolanmiu** [comment, marked as answer]
Replying is not possible.

I did some research, and it is a Microsoft Word only feature. This will not work in other Word Processors
